pyeasee/easee.py

```diff
@@ -411,7 +411,12 @@ async def get_account_products(self) -> List[Site]:
         try:
             records = await (await self.get("/api/accounts/products")).json()
             _LOGGER.debug("Sites:  %s", records)
-            sites = await asyncio.gather(*[self.get_site(r["id"]) for r in records])
+            sites = []
+            for r in records:
+                site = await self.get_site(r["id"])
+                site["circuits"] = r["circuits"]
```

Comment:
Only add circuits with accessible chargers and only add the accessible chargers to that circuit.

Reply:
The circuits that are returned from the site API are here overwritten with what was returned by the products API r["circuits"].
So this should be what this code does already?
Or do I misunderstand your comment?